.github/workflows/macos_wheel.yaml

```diff
@@ -45,33 +83,28 @@ jobs:
       - name: Install PyTorch
         run: |
           python -m pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/cpu
-      - name: Check out repo
+      - name: Install torchcodec from the wheel
+        run: |
+          wheel_path=`find pytorch/torchcodec/dist -type f -name "*.whl"`
+          echo Installing $wheel_path
+          python -m pip install $wheel_path -vvv
+      - name: Check out torchcodec repo
         uses: actions/checkout@v3
-      - name: Install compile from source dependencies
+      - name: Install ffmpeg
         run: |
-          conda install cmake pkg-config -c conda-forge
+          conda install "ffmpeg=${{ matrix.ffmpeg-version-for-tests }}" -c conda-forge
+          ffmpeg -version
       - name: Install test dependencies
         run: |
           python -m pip install --pre torchvision --index-url https://download.pytorch.org/whl/nightly/cpu
-          # Ideally we would find a way to get those dependencies from pyproject.toml
           python -m pip install numpy pytest pillow
-      - name: Install torchcodec from source, building against non-GPL FFmpeg
-        run: |
-          BUILD_AGAINST_ALL_FFMPEG_FROM_S3=1 pip install -e ".[dev]" --no-build-isolation
-      - name: Install ffmpeg, post build
+      - name: Delete the src/ folder just for fun
         run: |
-          # Ideally we would have checked for that before installing the wheel,
-          # but we need to checkout the repo to access this file, and we don't
-          # want to checkout the repo before installing the wheel to avoid any
-          # side-effect. It's OK.
-          source packaging/helpers.sh
-          assert_ffmpeg_not_installed
-
-          conda install "ffmpeg=${{ matrix.ffmpeg-version-for-tests }}" -c conda-forge
-          ffmpeg -version
+          rm -r src/
```

Comment:
Nit: either copy the comment made in the linux file https://github.com/NicolasHug/torchcodec/blob/d43dd9120d227ad922079830c05b3c8fa3662de4/.github/workflows/linux_wheel.yaml#L110-L119 or just point to it, to explain why we do this?

Once we merge this PR and when we have the CUDA wheel builds, it'll be time to refactor the 3 separate scripts into something more centralized